diff -ur sdrplay-orig/install_lib.sh sdrplay/install_lib.sh
--- sdrplay-orig/install_lib.sh	2020-09-03 10:59:09.760226641 +0100
+++ sdrplay/install_lib.sh	2020-09-03 11:01:16.763174084 +0100
@@ -4,33 +4,12 @@
 export MAJVERS="3"
 
 echo "Installing SDRplay RSP API library ${VERS}..."
-read -p "Press RETURN to view the license agreement" ret
 
-more sdrplay_license.txt
-
-while true; do
-    echo "Press y and RETURN to accept the license agreement and continue with"
-    read -p "the installation, or press n and RETURN to exit the installer [y/n] " yn
-    case $yn in
-        [Yy]* ) break;;
-        [Nn]* ) exit;;
-        * ) echo "Please answer y or n";;
-    esac
-done
-
-export ARCH=`uname -m`
+export ARCH=aarch64
 
 echo "Architecture: ${ARCH}"
 echo "API Version: ${VERS}"
 
-if [ "${ARCH}" != "aarch64" ]; then
-	if [ "${ARCH}" != "arm64" ]; then
-		echo "The architecture on this device (${ARCH}) is not currently supported."
-		echo "Please contact software@sdrplay.com for details on platform support."
-		exit 1
-	fi
-fi
-
 echo "If not installing as root, you will be prompted for your password"
 echo "for sudo access to the /usr/local area..."
 sudo mkdir -p /usr/local/lib >> /dev/null 2>&1
